zookeeper-contrib/zookeeper-contrib-loggraph/pom.xml

```diff
@@ -33,6 +33,9 @@
   <description>
     LogGraph is an application for viewing and filtering zookeeper logs. It can handle transaction logs and message logs.
   </description>
+  <properties>
+    <jetty.version>9.4.14.v20181114</jetty.version>
```

Comment:
Latest jetty is 9.4.15.v20190215 - any reason we wouldn't use that?

Reply:
It is the same version we have in master
@phunt

Reply:
Should we update both of them? :-)

Reply:
Good.
I will update this patch and create an issue for master and 3.5